docker-compose.yml

```diff
@@ -24,8 +23,6 @@ services:
     ports:
       - 0.0.0.0:1234:1234
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost"]
+      test: ["CMD", "curl", "-f", "http://localhost:1234"]
```

Comment:
Oh, good catch!

Reply:
I don't mind adding curl for this, but any idea if we can do this without adding the curl dependency? I'm not opposed if this is the most straight-forward, just thinking out loud.

Reply:
The default docker image node-alpine doesn't contain cli tool for checking, so any tool like curl/wget is usually added with.
or If we don't use healthcheck result,  we may delete healthcheck setting and curl.

Reply:
Alright, I think keeping the healthcheck and curl are good 👍